src/i18n/translation.fr.js

```diff
@@ -3,7 +3,7 @@ export default {
     headerTitle: 'COVID-19: Ce que vous devez savoir',
     noResultsMessage: 'Aucun résultat trouvé pour cet URL',
     changeAudience: 'Refaire le questionnaire',
-    audiencePrefix: "Les informations ci-dessous s'appliquent à "
+    audiencePrefix: "Les informations ci-dessous s'appliquent "
```

Comment:
Should have a colon, no?

Reply:
I thought that in french colons are not grammarly correct?